backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java

```diff
@@ -165,10 +165,18 @@ public Person updatePatient(
             employedInHealthcare
         );
 
-        if (patientToUpdate.getRole() != PersonRole.STAFF) {
-            Facility facility = _os.getFacilityInCurrentOrg(facilityId);
-            patientToUpdate.setFacility(facility);
+        Facility facility = null;
```

Comment:
This logic is duplicated in `addPatient` and needs to be updated there as well line: 115